code/captureImagery.py

```diff
@@ -0,0 +1,65 @@
+# Data capture
+import os
+import webcam
+import datetime, time
+import sys
+
+# Initialize webcam
+image_capture = webcam.webcam()
+# Sleep to let the webcam initialize
+time.sleep(.1)
+
+# Define function for capturing datasets using webcam
+def capture_image_set(number_of_images, type):
+	if type == "kitchen":
```

Comment:
think of using a list for type check. 
if type in ["kitchen", "living_room", "dining_room"]